### PR TITLE
WildFly mini conference recap

### PR DESCRIPTION
**Pull request by hpehl**
This PR adds the following changes: 

- removes the banner from the homepage
- adds a news post about the WildFly mini-conference
- changes to the conference page

A preview is available at https://hpehl.github.io/wildfly.org/

### CODE REVIEW

File: _posts/2024-03-19-WildFly-mini-conference-recap.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "WildFly Mini Conference"
```
Comment (fjuma) [inline _posts/2024-03-19-WildFly-mini-conference-recap.adoc:3]: Maybe we could use "WildFly Mini Conference Recap" as the title? WDYT?

File: _posts/2024-03-19-WildFly-mini-conference-recap.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+description: WildFly Mini Conference
+---
+
+= WildFly Mini Conference
+
+Many thanks to all participants of the first WildFly Mini Conference!
+
+Our first WildFly Mini Conference took place on March 6. We had four talks on the https://www.wildfly.org/conference/[agenda] and many interesting questions and feedback. The feedback we have received so far has been very positive and has encouraged us to organize similar events for the community in the future. Stay tuned!
+
+If you haven't done so far, please let us know how you liked the conference using our https://forms.gle/ruEytViEBBwthr5K7[feedback form].
```
Comment (fjuma) [inline _posts/2024-03-19-WildFly-mini-conference-recap.adoc:16]: Maybe after this sentence, we could mention that if anyone missed the conference, they can watch the recording?

**Comment by fjuma** [comment]
Looks great @hpehl! Thanks very much for working on this. I've just added a couple small comments.

**Review by fjuma — APPROVED** [review]

File: _posts/2024-03-19-WildFly-mini-conference-recap.adoc (hunk 1 above)
Comment (hpehl) [reply _posts/2024-03-19-WildFly-mini-conference-recap.adoc]: TBH, I had "Recap" in the title in the first place, but I wasn’t sure if that was good English. Changing back to include "Recap"

File: _posts/2024-03-19-WildFly-mini-conference-recap.adoc (hunk 2 above)
Comment (hpehl) [reply _posts/2024-03-19-WildFly-mini-conference-recap.adoc:18]: I added a sentence about that.

**Pull request merged**